pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
Line 2 should read x multiplied by y.

Reply:
They put this is to trigger the error message I think. Mine has a random "z" variable in when the rest of it was x and y

Reply:
well spotted